### PR TITLE
Add init-db docs and auto-activate first user

### PR DESCRIPTION
**Pull request by schwarzkrieger**
Refs #1698

### CODE REVIEW

**Comment by schwarzkrieger** [comment]
I need to rework tcms.kiwi_auth.tests.test_views to handle the first user auto activation.

File: docs/source/installing_docker.rst [hunk 1]
```diff
@@ -68,6 +68,20 @@ configuration of a new Kiwi TCMS installation::
...
+registration page:
+``https://<your_domain_or_ip>/accounts/register/``
+
+.. note::
+
+    The first registered user is automatically granted superuser rights and
+    is activated without mail verification.
+
+The next step is :ref:`configure-kiwi-domain`.
+
```
Comment (atodorov) [inline docs/source/installing_docker.rst:84]: The sequence of actions and documentation doesn't make logical sense. Assuming you start with a fresh system you need the minimum amount of documentation in order to get the system up and running. Which already is:

1) web approach: Go to https://localhost (or FQDN) and follow the prompts or 

cli approach:
2) Use the initial_setup cli command which is comprised of migrate + createsuperuser + set_domain + refresh_permissions. <- the command which we want people to use is initial_setup. The rest should be mentioned without too much information. It is expected that if you are going to use them you know what they are. If you don't stick to initial_setup.

Notes:
refresh_permissions is documented in admin.rst
set_domain is documented in the next section

createsuperuser isn't documented from what I can see.

In addition all commands have a 1 line documentation in admin.rst as well so we don't have to repeat that too much.

File: tcms/kiwi_auth/forms.py [hunk 2]
```diff
@@ -47,6 +47,7 @@ def save(self, commit=True):
 
         if User.objects.filter(is_superuser=True).count() == 0:
             user.is_superuser = True
+            user.is_active = True
```
Comment (atodorov) [inline tcms/kiwi_auth/forms.py:50]: this will needs tests.

**Review by atodorov — CHANGES_REQUESTED** [review]

File: docs/source/installing_docker.rst [hunk 3]
```diff
@@ -68,22 +68,18 @@ configuration of a new Kiwi TCMS installation::
 
 .. versionadded:: 8.9
 
-Alternatively you can use more automation-friendly setup described below:
+If you prefer web-based tool you can use
+``https://<your_domain_or_ip>/init-db/``
```
Comment (atodorov) [inline docs/source/installing_docker.rst:72]: Add `.. versionadded:: 10.1`, see for syntax above

File: docs/source/installing_docker.rst [hunk 4]
```diff
@@ -132,6 +128,9 @@ To upgrade running Kiwi TCMS containers execute the following commands::
     docker-compose up -d
     docker exec -it kiwi_web /Kiwi/manage.py migrate
 
+The migrations may be performed by using the web-based tool:
+``https://<your_domain_or_ip>/init-db/``
+
```
Comment (atodorov) [inline docs/source/installing_docker.rst:133]: Remove this block. We don't want to recommend performing web based migrations for upgrades.

**Review by atodorov — CHANGES_REQUESTED** [review]
Minor changes requested.

Squash all commits afterwards b/c this will be ready to be merged.

**Pull request merged**